docs/plotdict.xml

```diff
@@ -3801,6 +3811,44 @@ end</example>
 				"Inventory autosort" setting in General Game Settings.
 			</description>
 		</command>
+		<command id="itemsreward">
+			<canon>items reward (number)</canon>
+			<shortname>items reward</shortname>
+			<description>
+				Returns the id numbers of the items dropped by enemies in the last battle
+				(-1 if item reward number is not present).
```

Comment:
"-1 if item reward <p>number</p> is not present" would be clearer.
Also whle the example is self-explanatoy, we should probably add something like "<p>number</p>counts from 0 up, where <tt>items reward(0)</tt> is the ID of the first gained item, etc."